renovate.json5

```diff
@@ -28,9 +20,5 @@
       packageNames: ['test'],
```

Comment:
shouldn't this be put in the top level config as well?

Reply:
I am not sure. The comment says it is a fake dependency used in tests, so this might be specific to this repository.
@eduardoboucas Is this correct?